### DISCUSSION TITLE
Boot looping problems at the instalation part HELPP

### DISCUSSION THREAD

**Discussion by lucazsh** [CodeRunner5235/Opencore-Steam-Deck#4] [Q&A]
Hi, can u help me? I have problems with my Mac os installer. firstly it freezes when I try to format the ssd from the Steam deck and secondly after I install it on the unformatted ssd it keeps restarting at the Apple logo and 29 minutes remaining. I also tried with other OS like MacOS Big Sur and now Somona! Do I have any Idea why this is happening? Thanks ☺️

**Comment by lucazsh** [comment]
you*

**Reply by CodeRunner5235** [reply]
Can you send you deck version (LCD or OLED, along with storage type) an your config.plist?

**Reply by lucazsh** [reply]
I have the steam deck lcd with 64 GB SSD. nvme steam deck
[config.plist.txt](https://github.com/user-attachments/files/17416388/config.plist.txt)

**Reply by CodeRunner5235** [reply]
MacOS does tend to restart a lot on installation, you can try leaving it for a few hours to see if it fixes itself first. 

That being said, it is possible that the eMMC that comes with the 64GB deck could be part of the issue, as it is, well, eMMC instead of a standard SSD. You can try a kext like EmeraldSDHC to see if that helps at all, but you may end up having to replace the internal drive with an actual SSD or install on an external drive over USB

**Reply by lucazsh** [reply]
Thank you very much! I will try this! ☺️

**Reply by lucazsh** [reply]
Hey @CodeRunner5235! Unfortunately it didn't work... It's stuck at Waiting for DMOS... after a successful instalation...

**Reply by lucazsh** [reply]
Is this maybe bc I didn't complete my PlatformInfo? With the serial number etc...

**Reply by CodeRunner5235** [reply]
That would do it. The dmos us “dont steal macOS” so if your platform info is messed up, that can cause that issue

**Reply by lucazsh** [reply]
@CodeRunner5235 Still the same... 😭😭

**Reply by lucazsh** [reply]
Now what else could be causing the problem?

**Comment by CodeRunner5235** [comment]
Give this a shot. The only difference is the smbios is MacBookPro16,3. 
[config.plist.txt](https://github.com/user-attachments/files/17446873/config.plist.txt)

**Reply by lucazsh** [reply]
Nothing has changed... @CodeRunner5235 Can u send your own efi folder that u know that worked?

**Comment by lucazsh** [comment]
Hey @CodeRunner5235  maybe the problem is that I installed it on a usb stick 3.0?

**Reply by CodeRunner5235** [reply]
I havent heard of macOS throwing a fit because its installed on a USB stick, but anything is possible. Everyone I know who has gotten it booting has been using external hard drives or the internal NVME
